pkg/iac/rego/embed_test.go

```diff
@@ -116,15 +116,15 @@ func Test_RegisterDeprecatedRule(t *testing.T) {
 	}{
 		{
 			name: "deprecated check",
-			id:   "AVD-DEP-0001",
+			id:   "DEP-0001",
 			inputPolicy: `# METADATA
 # title: "deprecated check"
 # description: "some description"
 # scope: package
 # schemas:
 # - input: schema["dockerfile"]
 # custom:
-#   avd_id: AVD-DEP-0001
+#   id: DEP-0001
```

Comment:
Maybe I’m missing something, so please feel free to correct me.

For now, we only mark the `AVDID` field as deprecated.
However, it will still be used by users for some time (until it is fully removed).
Therefore, we should first check `id`, then `AVDID` for backward compatibility (common logic).
Tests should also keep both values.